defaults/main.yml

```diff
@@ -40,6 +40,10 @@ kubernetes_apt_repository: "deb http://apt.kubernetes.io/ kubernetes-xenial {{ k
 kubernetes_apt_ignore_key_error: false
 
 kubernetes_yum_arch: x86_64
+kubernetes_yum_base_url: "https://packages.cloud.google.com/yum/repos/kubernetes-el7-{{ kubernetes_yum_arch }}"
```

Comment:
Can you also add these in the README.md so they are documented there? After that I'm happy to merge!

Reply:
Done, please review again